docker-compose.yml

```diff
@@ -22,7 +23,7 @@ services:
       dockerfile: Dockerfile.trellis-ext-db
     environment:
       COGNITO_USER_POOL_ID: ${COGNITO_USER_POOL_ID:-us-west-2_CGd9Wq136}
-      AWS_COGNITO_REGION: ${AWS_COGNITO_REGION:-us-west-2}
+      AWS_REGION: ${AWS_COGNITO_REGION:-us-west-2}
```

Comment:
i think if you were to rename this env var, you'd also have to change the corresponding usage in trellis' `config.yml`:
https://github.com/LD4P/sinopia_server/blob/7df012ae98a8cc3418065a563ad0001dbd4f0a7a/trellis/etc/config.yml#L86

i guess you'd also want to build and push a new docker image?

no objection to the rename though, esp if you're setting up for some specific non-cognito re-use of the value -- it seems unlikely that this application will run different services in multiple regions, at least for now, so i imagine one region string is good enough 🙂

Reply:
@jmartin-sul @rsmith11 @jermnelson OK, here's the issue: it seems we have both `AWS_REGION` and `AWS_COGNITO_REGION` in our codebases:

* `AWS_REGION`: https://github.com/search?q=org%3ALD4P+AWS_REGION&type=Code & https://github.com/search?q=org%3Asul-dlss+AWS_REGION&type=Code
* `AWS_COGNITO_REGION`: https://github.com/search?q=org%3ALD4P+AWS_COGNITO_REGION&type=Code & https://github.com/search?q=org%3Asul-dlss+AWS_COGNITO_REGION&type=Code

Here's how it breaks down: `AWS_REGION` is used in `sinopia_acl` and all over Terraform. `AWS_COGNITO_REGION` is used in `sinopia_editor`, `sinopia_server`, and in Terraform but only in the `sinopia_editor` context.

The question then—for @jermnelson and @rsmith11, I suppose—is whether we **want** to distinguish between the overall AWS region and the AWS region for Cognito. If not, it would be ideal to use `AWS_REGION` consistently everywhere.

Reply:
@jmartin-sul we will use `AWS_REGION` across the board, per @jermnelson today. I have amended this PR and will be rolling out a series of PRs that support this work. I think this is ready for another look.